spring-native/storage-blob-native/README.md

```diff
@@ -154,10 +154,10 @@ NOTE: If you want to build a lightweight container containing a native executabl
 
 Add the `spring.cloud.azure.storage.blob.account-key` configuration, and replace the relevant values in *application.yml* according to the saved output variable value. You can find these values in the temp file *terraform/terraform.tfstate*, or you can visit the Azure portal to get them.
 
-Run `mvn -Pbuildpack package spring-boot:build-image`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application) for more details.
+Run `mvn -P spring-cloud-azure-4.x,buildpack-4.x package spring-boot:build-image`, see [Build the native application](https://docs.spring.io/spring-native/docs/current/reference/htmlsingle/#_build_the_native_application) for more details.
```

Comment:
It's better to add these contents in `spring-native/README.md`.

Reply:
The file `spring-native/README.md` is used to make the root directory display only one level of directory structure, such as `spring-native`, not `spring-native/storage-blob-native`.